ci-operator/step-registry/hypershift/mce/kubevirt/baremetalds/disconnected/conformance/hypershift-mce-kubevirt-baremetalds-disconnected-conformance-workflow.yaml

```diff
@@ -0,0 +1,61 @@
+workflow:
+  as: hypershift-mce-kubevirt-baremetalds-disconnected-conformance
+  documentation: |-
+    The HyperShift KubeVirt conformance workflow executes tests against a new ephemeral
+    HyperShift cluster. Administrative access to the control plane is provided
+    via the `KUBECONFIG` environment variable.
+    This is executed in a fully disconencted cluster with
+    mirrored images.
+
+    Note that tests will execute once the control plane is ready, regardless of
+    whether nodes have been requested, have joined, or are available.
+
+    The control plane is hosted on a dedicated standalone OCP cluster running
+    HyperShift in Azure account and is destroyed after tests complete.
+
+    The HyperShift launch capability is currently supported by the HyperShift
+    team.
+
+    Learn more about HyperShift here: https://github.com/openshift/hypershift
+
+    Track HyperShift's development here: https://issues.redhat.com/projects/HOSTEDCP
+  steps:
+    allow_best_effort_post_steps: true
+    allow_skip_on_success: true
+    post:
+    - ref: hypershift-mce-dump
+    - chain: gather-core-dump
+    - chain: hypershift-mce-kubevirt-destroy
+    - chain: baremetalds-ipi-post
+    test:
+    - chain: hypershift-conformance
+    pre:
+    - chain: baremetalds-ipi-pre
+    - ref: hypershift-mce-install-disconnected
+    - chain: hypershift-kubevirt-baremetalds-metallb
+    - chain: hypershift-kubevirt-baremetalds-odf
+    - ref: hypershift-kubevirt-install-disconnected
+    - chain: hypershift-mce-kubevirt-create-disconnected
+    - chain: hypershift-mce-kubevirt-baremetalds-proxy
+    env:
+      TEST_SKIPS: etcd leader changes are not excessive
+      METALLB_OPERATOR_SUB_SOURCE: mirror-catalog-for-olm
+      LOCAL_STORAGE_OPERATOR_SUB_SOURCE: mirror-catalog-for-olm
+      ODF_OPERATOR_SUB_SOURCE: mirror-catalog-for-olm
+      DISCONNECTED: "true"
+      ENABLE_ICSP: "true"
+      DEVSCRIPTS_CONFIG: |
+        IP_STACK=v4
+        MIRROR_IMAGES=true
+        INSTALLER_PROXY=true
+        MIRROR_OLM="local-storage-operator,metallb-operator,odf-operator,ocs-operator,mcg-operator,odf-csi-addons-operator"
+        NETWORK_TYPE=OVNKubernetes
+        NUM_WORKERS=0
+        NUM_MASTERS=3
+        MASTER_VCPU=16
+        MASTER_MEMORY=81920
+        VM_EXTRADISKS=true
+        VM_EXTRADISKS_LIST=vda
+        VM_EXTRADISKS_SIZE=1024G
+        IPv6=no
+        IPv4=yes
```

Comment:
These two lines are redundant, we can move them.

Reply:
done